data/area/karamja/brimhaven/brimhaven.objs.toml

```
@@ -21,3 +21,43 @@ examine = "I hope I don't fall off this."
 [brimhaven_log_balance_end]
 id = 5090
 examine = "I hope I don't fall off this."
+
+[brimhaven_dungeon_entrance]
+id = 5083
+examine = "An overgrown dungeon entrance."
+
+[brimhaven_dungeon_exit]
+id = 5084
+examine = "A closed overgrown dungeon entrance"
+
+[brimhaven_walk-up_1]
```

```suggestion
[brimhaven_walk_up_1]
```

Underscores only please